plugin/black.vim

```diff
@@ -37,10 +37,17 @@ if !exists("g:black_skip_string_normalization")
 endif
 
 python3 << endpython3
+import os
 import sys
 import vim
 
 def _get_python_binary(exec_prefix):
+  try:
+      default = vim.eval("g:pymode_python").strip()
+  except:
```

Comment:
Pokemon-style "except:" is an anti-pattern. Can you catch the actual exception you are expecting?

Reply:
Thank you for the suggestion, I have fixed this in my latest commit to this PR.